xonsh/prompt/cwd.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Created by i@BlahGeek.com at 2016-09-03
```

Comment:
We don't have a custom of signing files in xonsh. If you could remove this line, that would be most appreciated!